deepmd/pt/model/descriptor/repflow_layer.py

```
@@ -0,0 +1,938 @@
+
+        Returns
+        -------
+        n_updated:     nf x nloc x n_dim
+            Updated node embedding.
+        e_updated:     nf x nloc x nnei x e_dim
+            Updated edge embedding.
+        a_updated : nf x nloc x a_nnei x a_nnei x a_dim
+            Updated angle embedding.
+        """
+        nb, nloc, nnei, _ = edge_ebd.shape
+        nall = node_ebd_ext.shape[1]
```

Ruff (0.8.2)

Local variable `nall` is assigned to but never used

Remove assignment to unused variable `nall`

(F841)